### ISSUE TITLE
[INFO] New Docker Images

### ISSUE THREAD

**Issue by h44z** [h44z/wg-portal#191]
## New Docker-Hub repository
The Docker-Hub repository was moved to an organization: https://hub.docker.com/r/wgportal/wg-portal.

Please update all old image references from **h44z/wg-portal** to **wgportal/wg-portal.**

## Updated versioning schema

There are the following new tags:

* **latest**: The latest build from the master branch. Not a stable tag!
* **master**: same as latest
* **stable**: The latest build from the stable branch.
* **v1**: The latest version of WireGuard Portal V1
* **v2**: The latest version of WireGuard Portal V2
* **major.minor**: Latest x.y version (available for version 1 and 2)
* **major.minor.patch**: Exact x.y.z version, corresponding to the Git tag (available for version 1 and 2)

**Comment by TheAlbanian** [comment]
hello i install from docker basic like you post only i add wgportal/wg-portal:master 
replace to wgportal/wg-portal:v2 i no edit more.

my question are wg0.conf no created in - /etc/wireguard

Generation of wg-quick configuration file (wgX.conf) if required

i search all in my debian 
find –name / wg0.conf
find: 'wg0.conf': No such file or directory

pls can make more details in README ...

maybe config are in db... even i want put linux firewall settings ...i think no work

**Comment by bonddim** [comment]
@h44z did you though about transferring repo to *wgportal* org ?
I see that such name is available now.

**Comment by PizzaProgram** [comment]
Now, that V2 is basically ready to use, is:
 `:latest` now the same as `:v2` ?

**Comment by h44z** [comment]
@PizzaProgram 

- **latest** is always the latest commit in the master branch
- **v2** is always the latest release of version 2 (including beta and release-candidate releases)

So there might be a difference between **latest** and **v2** - latest is never considered to be stable, you should use **v2** instead.

I have just released v2.0.0-rc2. If there are no more error reports over the weekend, the final stable version of v2 will be released next week.

**Comment by mjbogusz** [comment]
Since 2.0 was released 🍾🎆, could you clarify what's the difference between tags `2`, `v2` and `stable`?

I'm asking because `v2` mentioned in the docs is the same as `v2.0.0-rc.4`, `2` is `v2.0.0` and `stable` is yet another different version. I'm kinda assuming that some tags were just lost in the release process, but I'd rather be sure (:

**Comment by h44z** [comment]
@mjbogusz, you are right. The images in the GitHub container registry are messed up. I just fixed them.

In general, the following rules apply:

- all versioned tags are available with and without the '**v**' prefix (e.g: `v2.0.0` and `2.0.0`)
- `v2` (or just `2`) is always the latest image for WireGuard Portal version 2
- `v2.x` (or just `2.x`) is always the latest minor release of WireGuard Portal version 2.x
- `v2.x.y` (or just `2.x.y`) denotes a specific release. Such an image will always stay the same.
- `master`, `stable`, and so on are images built from a specific git branch, they should not be considered official releases (even though the `stable` image will mostly match the `v2` image.

To summarize: if you want a stable experience, use one of the `v2` tags. Always use the `v2.x.y` tag if you require an explicit version.
If you are feeling adventurous, you can use the `master` image to test out new features before they are finally released.

If you have any suggestions on how to improve the versioning scheme, so that it is easier to understand, let me know =)
